javascript/javascript1/week1/Housey-pricey.js

```diff
@@ -0,0 +1,17 @@
+let deep = prompt('enter the deep/length value in meters')
```

Comment:
Nice solution with using the `prompt` to avoid writing the same code for both Julia and Peter. :star_struck:

Reply:
thanks